torch_glow/tests/nodes/reciprocal_test.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch_glow
+from tests.utils import jitVsGlow
+
+
+def test_reciprocal():
+    """Test of the PyTorch reciprocal Node on Glow."""
+
+    def reciprocal(a):
+        c = torch.reciprocal(a)
+        return torch.reciprocal(c)
```

Comment:
It would be better to use another op in here like `return torch.reciprocal(a+a)` because test can't distinguish between reciprocal of reciprocal and noop.